### PR TITLE
Add tasks for checker tools

### PR DESCRIPTION
**Pull request by therunnergit**
This PR adds a public task to run some code checker tools, namely `phpmd`, `phpcs` and `phpcpd`. 
Later on, if we refactor the Robofile.php, they will probably be moved outside it and into the shared tasks.

# Usage
To execute any of the given tool, you execute:

    vendor/bin/robo run:checker [checkername]

e.g.

    vendor/bin/robo run:checker phpmd

# How to test

Run the following commands and make sure you get some result:

    vendor/bin/robo run:checker phpmd
    vendor/bin/robo run:checker phpcs
    vendor/bin/robo run:checker phpcpd

# Notes

* The checks are done against the `src/` folder.
* The result will be an error because we DO have code issues to fix, and so the exit code is not 0.

### CODE REVIEW

**Comment by Kubik-Rubik** [comment]
Can you please explain in more detail how to test it with one of the checker tools?

**Comment by therunnergit** [comment]
@Kubik-Rubik I updated the original description with more details.

File: RoboFile.php [hunk 1]
```diff
@@ -156,6 +156,30 @@ public function runTest($pathToTestFile = null, $suite = 'acceptance')
 		     ->stopOnFail();
 	}
 
+	public function runChecker($tool = null)
```
Comment (javigomez) [inline RoboFile.php]: missing fuction comment and parameter description. Is needed for when you list the documentation:

screen shot 2015-10-31 at 11 36 57

And also when you look for the parameters help:

screen shot 2015-10-31 at 11 37 57

**Comment by Kubik-Rubik** [comment]
@therunnergit This is great, thank you for the explanation and improvement. Works as described!

**Comment by therunnergit** [comment]
@javigomez improved documentation and used \Robo\Result instead of straight exit codes.

**Comment by Kubik-Rubik** [comment]
Works with the last change properly!

**Comment by nn-medienagentur** [comment]
@Test running fine. Awesome work!

**Comment by Kubik-Rubik** [comment]
Thank you @therunnergit! Merged.

**Pull request merged**